config/crds/tenancy.kcp.dev_clusterworkspacetypes.yaml

```diff
@@ -53,37 +53,83 @@ spec:
                 description: additionalWorkspaceLabels are a set of labels that will
                   be added to a ClusterWorkspace on creation.
                 type: object
+              allowAnyChildWorkspaceTypes:
+                description: allowAnyChildWorkspaceTypes permits all ClusterWorkspaceTypes
+                  to be created in a workspace of this type.
+                type: boolean
+              allowAnyParentWorkspaceTypes:
```

Comment:
I like this. It allows us sensible defaults (through admission 🤷‍♂️). Just a sum type would make this nicer:
```
allowedChildren:
  any: {}
  types: [{path: ..., name: ...}]
```